lib/stringex/string_extensions.rb

```diff
@@ -187,8 +187,8 @@ def to_url(options = {})
         whitespace_replacement_token = options[:replace_whitespace_with]
         dummy = remove_formatting(options).
                   replace_whitespace(whitespace_replacement_token).
-                  collapse("-").
-                  limit(options[:limit], options[:truncate_words])
+                  collapse(whitespace_replacement_token).
+                  limit(options[:limit], options[:truncate_words], whitespace_replacement_token)
```

Comment:
Line is too long. [96/80]<br>Align the operands of an expression in an assignment spanning multiple lines.